Framework/AODMerger/src/aodThinner.cxx

```diff
@@ -337,6 +425,24 @@ int main(int argc, char* argv[])
     gSystem->Unlink(outputFile->GetName());
   }
 
+  clock.Stop();
+
+  try {
```

Comment:
is the try except really needed here? If yes, maybe a simple gFile function to get the file size?

Reply:
dropped